### PR TITLE
WIP: Upgrade to CsvHelper v22

### LINKED ISSUE #226: Deprecation of IgnoreQuotes in CsvHelper v 20.0.0
**Describe the bug**
The IgnoreQuotes parameter has been removed from CsvHelper as of v20.0.0 - need to find either a workaround to preserve this functionality, or document and reflect this removal.

**Additional context**
Unknown: how important is this option in actual RDMP usage? Would removing it upset people enough that it needs to be re-implemented somehow?

### PR DESCRIPTION
**Pull request by rkm**
Fixes #226.

Notes:
-   Will require a new release of BadMedicine

### CODE REVIEW

**Comment by jas88** [comment]
> * Will require a new release of BadMedicine

PR #258

File: Rdmp.Core/DataLoad/Modules/DataFlowSources/DelimitedFlatFileDataFlowSource.cs [hunk 1]
```diff
@@ -391,22 +391,21 @@ protected void OpenFile(FileInfo fileToLoad)
             {
                 Delimiter = Separator,
                 HasHeaderRecord = string.IsNullOrWhiteSpace(ForceHeaders),
-                ShouldSkipRecord = ShouldSkipRecord
+                ShouldSkipRecord = ShouldSkipRecord,
+                IgnoreBlankLines =  IgnoreBlankLines,
+                Mode = IgnoreQuotes ? CsvMode.Escape : CsvMode.RFC4180,
```
Comment (tznind) [inline Rdmp.Core/DataLoad/Modules/DataFlowSources/DelimitedFlatFileDataFlowSource.cs:396]: Ignore Quotes might be CsvMode.NoEscape ?

Comment (tznind) [reply Rdmp.Core/DataLoad/Modules/DataFlowSources/DelimitedFlatFileDataFlowSource.cs:396]: Can you make sure that you have the test `Test_IgnoreQuotes` in your working copy and that it passes?

**Comment by rkm** [comment]
Thanks Thomas - this had dropped off my radar...

**Pull request closed without merging**